src/components/modular-content/index.njk

```diff
@@ -13,14 +13,16 @@
 
 {% macro modularContent(items, context) %}
 <div class="modular-content">
-  {% for item in items %}
-    {% set itemType = item._modelApiKey %}
-    {% set component = componentByType[itemType] %}
-    {% if component %}
-      {{ component(item, context) }}
-    {% else %}
-      <script>console.warn('Unknown item type "{{ itemType }}"');</script>
-    {% endif %}
-  {% endfor %}
+  <div class="container modular-content__inner">
```

Comment:
So there will never be any content that spans the full window width?

Reply:
there's no such a block in the design